docs/reference/commandline/run.md

```diff
@@ -65,6 +65,7 @@ parent = "smn_cli"
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
       --privileged=false            Give extended privileges to this container
+      --read-bps-device=""       Specifies upper limit on READ rate from the device
```

Comment:
Also, can this option be specified multiple times (for multiple devices?). If so, we should probably mention it as `--read-bps-device=[]` (square brackets)